ledger-api/grpc-definitions/com/daml/ledger/api/v2/transaction_filter.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v1/transaction_filter.proto";
+
+option java_outer_classname = "TransactionFilterOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
+// A filter both for filtering create and archive events as well as for
+// filtering transaction trees.
+message TransactionFilter {
```

Comment:
Sorry for the late question but why "TransactionFilter" ? We use it for other things: updates, acs